src/util/Constants.js

```diff
@@ -637,8 +637,11 @@ exports.VerificationLevels = createEnum(['NONE', 'LOW', 'MEDIUM', 'HIGH', 'VERY_
  * * UNKNOWN_REDISTRIBUTABLE
  * * UNKNOWN_GIFT_CODE
  * * UNKNOWN_GUILD_TEMPLATE
+<<<<<<< HEAD
  * * UNKNOWN_INTERACTION
  * * UNKNOWN_APPLICATION_COMMAND
+=======
+>>>>>>> 70b04688 (fix: fix bugs found in runtime)
```

Comment:
Git rebase is still here, lurking away

Reply:
I...whh...how?  I literally remember removing those exact likens in this rebase 🤔

Reply:
Sounds like the rebase needs a rebase 😃

Reply:
well this PR no longer changes Constants at all 😆 , so hopefully that's a good sign that rebase is gone